content/en/security/cspm/detection_rules.md

```diff
@@ -0,0 +1,89 @@
+---
+title: Managing CSPM Detection Rules
+kind: documentation
+aliases:
+  - /security_platform/cspm/configuration_rules
+  - /security/cspm/configuration_rules
+further_reading:
+  - link: "/security/cspm/getting_started"
+    tag: "Documentation"
+    text: Getting Started with CSPM
+  - link: "/security/cspm/custom_rules/"
+    tag: "Documentation"
+    text: Custom Rules
+  - link: "/security/cspm/frameworks_and_benchmarks/"
+    tag: "Documentation"
+    text: Findings Reports
+---
+
+{{< site-region region="gov" >}}
+<div class="alert alert-warning">
+Cloud Security Posture Management is not currently available in this site.
+</div>
+{{< /site-region >}}
+
+Cloud Security Posture Management (CSPM) [out-of-the-box detection rules][1] evaluate the configuration of your cloud resources and identify potential misconfigurations so you can immediately take steps to remediate. 
+
+The detection rules follow the same [conditional logic][2] as all Datadog Security detection rules. For CSPM, each rule maps to controls within one or more [compliance frameworks or industry benchmarks][4].
+
+CSPM uses the following rule types to validate the configuration of your cloud infrastructure:
+
+- [**Cloud configuration**][1]: These detection rules analyze the configuration of resources within your cloud environment. For example, the [Cloudfront distribution is encrypted][3] rule evaluates an AWS Cloudfront distribution's configuration for encrypted status.
+- [**Infrastructure configuration**][5]: These detection rules analyze your containers and Kubernetes clusters to find configuration issues, as defined in the popular CIS compliance benchmarks for Docker and Kubernetes. For example, the [/etc/default/docker file permissions are set to 644 or more restrictively][6] rule evaluates Docker file permissions running on a host.
+
+## Customize how your environment is scanned by each rule
```

Comment:
I have a note that we need to link/mention how to do this via API. Can you confirm which endpoint docs we should link to? Is it the [security_monitoring endpoint](https://docs.datadoghq.com/api/latest/security-monitoring/#update-an-existing-rule)?